src/SourceBuild/content/prep-source-build.sh

```diff
@@ -204,7 +204,7 @@ function BootstrapArtifacts {
 
   properties=( "/p:ArchiveDir=$packagesArchiveDir" )
   if [[ -n "$target_rid" ]]; then
-    properties+=( "/p:TargetRid=$target_rid" )
+    properties+=( "/p:PortableRid=$target_rid" )
```

Comment:
Thoughts on renaming the 'target-rid' script option to portable rid?  This could provide more clarity in what value to specify.

Reply:
As a user of this script, the flag being named "target rid" makes a bit more sense to me, since the value is supposed to be RID we are targeting. The fact that it better be portable to work is more of an implementation detail?

Though I can see there's a good reason to keep the script option consistent with the property name: it makes maintenance of this script easier.

Anyway, no strong feelings either way. Let me know which way you prefer.

Reply:
I can see why "target rid" makes sense in this context.  My concern is consistency between with the prep and build scripts along with underlying msbuild logic.  In other words "target rid" here doesn't mean the same thing as in the build script.  The target rid here must be a portable rid.  At a minimum the option's documentation needs to be updated to reference that it must be a portable rid.

Reply:
Done.